db/migrate/20150621174941_create_interviews.rb

```diff
@@ -0,0 +1,15 @@
+class CreateInterviews < ActiveRecord::Migration
+  def change
+    create_table :interviews do |t|
+      t.integer :job_id
+      t.string :title
+      t.integer :number
+      t.string :type
+      t.string :details
+      t.date :date
+      t.string :time_between
+      t.string  :questions
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.